releases.md

```diff
@@ -0,0 +1,105 @@
+# Tekton Release Cycles
+
+## Support Policy
+
+The Tekton project maintains three release branches for each project, created one every four months, which
+results in a overall support window of approximately one year for each of these releases.
+
+This approach allows Tekton to roughly align to the [Kubernetes release cycles][kube-releases] and 
+maintain **four** supported releases at any point in time.
+
+Tekton is made of a collection of projects, and each project may define its own release cadence, as long as
+this [support policy](#support-policy) is followed. 
+
+Examples:
+
+- A project may release on a monthly basis, and maintain a release branch for every fourth release
+- A project may release every four months, and maintain a release branch for the three most recent minor
+  releases
+
+## Release Numbers
+
+Tekton uses [semantic versioning][semantic] to version its own releases. Release numbers are in the format
+MAJOR.MINOR.PATCH `vX.Y`. The [support policy](#support-policy) applies to MAJOR and MINOR releases alike.
+
+## Release Tags and Branches
+
+Every time a Tekton project produces a release, a new tag `vX.Y.Z` is created, as well as a new branch that
+initially points to the git tag `vX.Y.Z`.
+
+- Supported releases: 
+    - branch name: `release-vX.N-lts`
+    - support window: until `release-vX.N+3`
+
+- Any other major or minor release:
+    - branch name: `release-vX.M`
+    - support window: until `release-vX.M+1`
+
+Once a release support window expires, the corresponding branch is deleted.
```

Comment:
Is it worth explicitly stating "The corresponding tag is never deleted."?

Reply:
Yes, I can add that.